code/control/control.cpp

```diff
@@ -1,12 +1,12 @@
 #include <iostream>
 
-constexpr unsigned int numbers[]{1000001, 1000002, 1000003, 1000004, 1000005, 1000006, 1000007, 1000008, 1000009};
+const unsigned int numbers[]{1000001, 1000002, 1000003, 1000004, 1000005, 1000006, 1000007, 1000008, 1000009};
```

Comment:
Even though the `constexpr` is fine here, it confused some students during my afternoon session. So I propose to simplify it to a `const`.